src/jsonrpc/miner_jsonrpc_sc.erl

```diff
@@ -28,12 +28,20 @@ handle_rpc(_, _) ->
 %%
 %% Internal
 %%
-format_sc_list(SCs) ->
+format_sc_map(SCs) ->
     {ok, Height} = blockchain:height(blockchain_worker:blockchain()),
-    ActiveIds = blockchain_state_channels_server:active_sc_ids(),
-    maps:fold(fun(_SCID, {SC, _Skew}, Acc) ->
-                      #{ expire_at_block := ExpireAt } = Json
-                      = blockchain_state_channel_v1:to_json(SC, []),
-                      [ Json#{ <<"is_active">> => lists:member(SC, ActiveIds),
-                               <<"expired">> => Height >= ExpireAt } | Acc ]
-              end, [], SCs).
+    ActiveSCIDs = maps:keys(blockchain_state_channels_server:get_actives()),
+    maps:fold(
+        fun(SCID, SC, Acc) ->
+            #{expire_at_block := ExpireAt} = Json = blockchain_state_channel_v1:to_json(SC, []),
+            [
+                Json#{
+                    <<"is_active">> => lists:member(SCID, ActiveSCIDs),
+                    <<"expired">> => Height >= ExpireAt
+                }
```

Comment:
is this the Erlang equivalent Map merging syntax to Elixir's:
```elixir
%{
  json |
  "is_active" => Lists.member(scid, active_scids),
  "expired" => height >= exire_at
}
```

Reply:
nvm, I've confirmed this; it's actually _more_ powerful than Elixir's map update shorthand syntax b/c there it's not supported to use that syntax if the key isn't already present in the map (only equivalent to `Json#{ <<"is_active">> := lists:member(SCID, ActiveSCIDs)}`) which is pretty cool!